src/set_theory/zfc.lean

```diff
@@ -696,9 +696,27 @@ def to_Set (p : Set.{u} → Prop) (A : Class.{u}) : Prop := ∃ x, ↑x = A ∧
 protected def mem (A B : Class.{u}) : Prop := to_Set.{u} B A
 instance : has_mem Class Class := ⟨Class.mem⟩
 
-theorem mem_univ {A : Class.{u}} : A ∈ univ.{u} ↔ ∃ x : Set.{u}, ↑x = A :=
+@[simp] theorem mem_univ {A : Class.{u}} : A ∈ univ.{u} ↔ ∃ x : Set.{u}, ↑x = A :=
 exists_congr $ λ x, and_true _
 
+@[simp] theorem univ_apply (x : Set.{u}) : univ.{u} x := trivial
+
+/-- A proper class is one that isn't equal to the coercion of any set. -/
+def is_proper (A : Class.{u}) : Prop := A ∉ univ.{u}
```

Comment:
Do we really need a definition for this? `A ∉ univ` is shorter than `A.is_proper`, and also defining a negated thing seems less useful than having a definition for the positive version and negating it.

Reply:
Yeah, you're right. I'll use `∉ univ` to state the results I want to prove.